src/sagemaker/serve/builder/hf_dlc_builder.py

```diff
@@ -0,0 +1,208 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You don't need to amend commits here, since without CR-revision feat, it's a bit hard for me to track what's changed since last review. Also, multiple commits are totally fine since MLF team will eventually squash all of them and merge

Reply:
Yeah getting used to this tool and testing, will definitely follow this for the next PR.